thermal/thermal-daemon/thermal-conf.xml

```diff
@@ -281,61 +281,74 @@
+
 	<Platform>
-		<Name> APL-NUC </Name>
-		<ProductName>NUC6CAYH</ProductName>
-		<Preference>QUIET</Preference>
-		<ThermalZones>
-			<ThermalZone>
-				<Type>cpu_passive</Type>
-				<TripPoints>
-					<TripPoint>
-						<SensorType>x86_pkg_temp</SensorType>
-						<Temperature>90000</Temperature>
-						<Type>Passive</Type>
-						<CoolingDevice>
-							<Type>rapl_limit_1</Type>
-						</CoolingDevice>
-					</TripPoint>
-					<TripPoint>
-						<SensorType>x86_pkg_temp</SensorType>
-						<Temperature>100000</Temperature>
-						<Type>Passive</Type>
-						<CoolingDevice>
-							<Type>rapl_limit_2</Type>
-						</CoolingDevice>
-					</TripPoint>
-				</TripPoints>
-			</ThermalZone>
-			<ThermalZone>
-				<Type>cpu_critical</Type>
-				<TripPoints>
-					<TripPoint>
-						<SensorType>x86_pkg_temp</SensorType>
-						<Temperature>104000</Temperature>
-						<Type>Critical</Type>
-					</TripPoint>
-				</TripPoints>
-			</ThermalZone>
-		</ThermalZones>
+	 <Name> AlderLake-P </Name>
+                <ProductName>*</ProductName>
+                <Preference>QUIET</Preference>
+                <ThermalZones>
+                        <ThermalZone>
+                                <Type>cpu_passive</Type>
+                                <TripPoints>
+                                        <TripPoint>
+                                                <SensorType>x86_pkg_temp</SensorType>
+                                                <Temperature>75000</Temperature>
+                                                <Type>Passive</Type>
+                                                <CoolingDevice>
+                                                        <Type>TFN1</Type>
+                                                </CoolingDevice>
+                                        </TripPoint>
+                                         <TripPoint>
+                                                <SensorType>x86_pkg_temp</SensorType>
+                                                <Temperature>83000</Temperature>
+                                                <Type>Passive</Type>
+                                                <CoolingDevice>
+                                                        <Type>rapl_limit_1</Type>
+                                                </CoolingDevice>
+                                        </TripPoint>
+                                        <TripPoint>
+                                                <SensorType>x86_pkg_temp</SensorType>
+                                                <Temperature>90000</Temperature>
+                                                <Type>Passive</Type>
+                                                <CoolingDevice>
+                                                        <Type>rapl_limit_2</Type>
+                                                </CoolingDevice>
+                                        </TripPoint>
+                                </TripPoints>
```

Comment:
does line no. 317, 318 need to be there?

Reply:
yes, its better to separate critical temperature trip points.